### PR TITLE
changes vampire hypnotize to be a sleep stun but makes it cost blood or something i guess this is supposed to be better since its prebase code

### PR DESCRIPTION
**Pull request by SomeguyManperson**
new vampire abilities:
Gaze: no cost, instantly stuns an adjacent target for 5 seconds, cooldown of 30 seconds
Hypnotize: costs 20 blood, 4 second delay, stuns and mutes target for 30 seconds if the target is still within 3 meters (subject to change)

:cl:  
fix: fixes some stupid action button things like blood 101 not having an icon and heretic background not having a name
fix: some spelling stuff in how to suck blood 101
rscadd: vampires now get two abilities, glare and hypnotize. Gaze is an instant short stun while hypnotize is a delayed longer stun-mute that costs blood to use. Neither work through flashproofing.
/:cl:

### CODE REVIEW

**Review by Partheo — APPROVED** [review]

**Comment by Marmio64** [comment]
people who wear sunglasses are immune?

**Comment by Xoxeyos** [comment]
I think it should go through flash proofing, a vampire is literally a creature that is **super natural**, and I believe that making it flash proof will make it way harder for the vampire to start off; I shouldn't have to rely on a stun prod to catch my prey.

**Comment by ghost** [comment]
it is stopped by flashproofing so that you don't instantly gamer a security officer

'muh super natural' is not an argument @Xoxeyos

File: yogstation/code/game/gamemodes/vampire/vampire_powers.dm [hunk 1]
```diff
@@ -115,31 +115,123 @@
...
+	action_icon = 'yogstation/icons/mob/vampire.dmi'
+	action_background_icon_state = "bg_demon"
+
+/obj/effect/proc_holder/spell/pointed/gaze/can_target(atom/target, mob/user, silent)
+	. = ..()
+	if(!.)
+		return FALSE
+	if(!isliving(target))
+		return FALSE
+	if(!ishuman(target))
```
Comment (Amelia0010) [inline yogstation/code/game/gamemodes/vampire/vampire_powers.dm:134]: Is there any reason not to just condense these two checks?

File: yogstation/code/game/gamemodes/vampire/vampire_powers.dm [hunk 2]
```diff
@@ -115,31 +115,123 @@
...
+		usr.visible_message("<span class='warning'>[usr] twirls their finger in a circlular motion.</span>",\
+				"<span class='warning'>You twirl your finger in a circular motion.</span>")
+	..()
+
+/obj/effect/proc_holder/spell/pointed/hypno/can_target(atom/target, mob/user, silent)
+	if(!..())
+		return
+	if(!isliving(target))
+		return FALSE
+	if(!ishuman(target))
```
Comment (Amelia0010) [inline yogstation/code/game/gamemodes/vampire/vampire_powers.dm:195]: condense these two checks as well

File: yogstation/code/game/gamemodes/vampire/vampire_powers.dm [hunk 3]
```diff
@@ -115,31 +115,123 @@
...
+					"<span class='warning'>[user]'s eyes flash red.</span>")
+	if(istype(target, /mob/living/carbon/human))
+		var/mob/living/carbon/human/T = target
+		var/obj/item/clothing/glasses/G = T.glasses
+		if(G)
+			if(G.flash_protect)
+				to_chat(user, "<span class='warning'>[T] has protective sunglasses on!</span>")
+				to_chat(target, "<span class='warning'>[user]'s paralyzing gaze is blocked by [G]!</span>")
+				return
+		var/obj/item/clothing/mask/M = T.wear_mask
```
Comment (Amelia0010) [inline yogstation/code/game/gamemodes/vampire/vampire_powers.dm:217]: Check the helmet as well so you don't get got through spacesuit helmets

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

**Comment by Tipy1802** [comment]
Yeah hypnosis beeing free is really op, good change

**Comment by Nechronom** [comment]
Maybe give gaze a very short cast time, so you can't just grab a rando and stuff him in a box during that split second. Well. you still could, but not if they're actively moving

**Review by Amelia0010 — APPROVED** [review]

**Pull request merged**